src/main/java/org/eyeseetea/sdk/common/DatabaseUtils.java

```diff
@@ -0,0 +1,40 @@
+package org.eyeseetea.sdk.common;
+
+
+import java.io.File;
+
+public class DatabaseUtils {
```

Comment:
this class should receive as a parameter on methods where is needed the context and database name directly without invoking other class for retrieving it as on FileUtils class.